ministryofjustice/correspondence_tool_staff
# CT-2915 add recipent detail

## Description

Add the step of allowing use to choose who will receive the information

This code is based on branch CT-2928-add-new-fields-for-contact-details
## Self-review checklist


### Screenshots


### Related JIRA tickets

https://dsdmoj.atlassian.net/browse/CT-2915
### Deployment


### Manual testing instructions

login as branston office
click create cases and choose offender sar
once reach the step of filling the detail of recipient, you should see the screenshots being attached in the ticket depending whether this case is raised by representative or data subject themselves
if you choose third party,  the validation around third party will be checked
